packages/sol-tracing-utils/src/artifact_adapters/truffle_artifact_adapter.ts

```diff
@@ -70,6 +79,16 @@ export class TruffleArtifactAdapter extends AbstractArtifactAdapter {
             );
         }
     }
+    private _getTruffleSolcSettings(): Partial<solc.CompilerSettings> {
+        const truffleConfig = this._getTruffleConfig();
```

Comment:
Please add add a comment here that this logic is here to support two versions of truffle's config file. Otherwise the next dev will be scratching his head.